### DISCUSSION TITLE
How can I change the listing price of an NFT ?

### DISCUSSION THREAD

**Discussion by Stanley-GF** [ProjectOpenSea/seaport#637] [Q&A] [answered]
What's up ? 
I'd like to know, how can I change the price (or transfer an nft) knowing that I only have these 4 informations:

- The receiver (`0xEc2d1Cb533F2FA3a753f7370086409dc106aFD37`)
 - The signature hash (`0x5dbc6c667e5eccaabffc725e6766d4e4cbbc26a68c19866122831972efe33cdd1bfa424c99ca4962469da29e8a8c3d75d6ef9478b162a855abe2f7696776f6931b`)
- The salt (`4125713862951889991818813470787979299798913853469635383343454919537990`, I highly doubt that that's useful)
- The NFT information (`[{"itemType":2,"token":"0x1485297e942ce64e0870ece60179dfda34b4c625","identifierOrCriteria":"2924","startAmount":"1","endAmount":"1"}]`)

I'd like to do it in NodeJS, if that's possible, or any other language except Solidity.

Thanks
- Stan

**Comment by ryanio** [comment, marked as answer]
If you want to change the order params you have to create and sign a brand new order.
